docs/authentication.md

```diff
@@ -25,5 +25,26 @@ public class Example {
 
 ```
 
+## Extension app authentication
+
+In order to authenticate your extension app, you need to have a proper `app` field defined in your configuration,
+see [authentication](authentication.md) for more information. We support both RSA and certificate authentication.
+
+With a [`SymphonyBdk`](../symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java) instance,
```

Comment:
In [another context](https://symphonyplatformsolutions.github.io/symphony-api-client-java/configuration.html) than Github repo this linking won't work. Instead you should link https://javadoc.io/doc/com.symphony.platformsolutions/symphony-bdk-core/latest/com/symphony/bdk/core/SymphonyBdk.html